server/command.go

```diff
@@ -320,6 +321,14 @@ func createWebhookCommand(optInstance bool) *model.AutocompleteData {
 	return webhook
 }
 
+func createSetupCommand(optInstance bool) *model.AutocompleteData {
+	setup := model.NewAutocompleteData(
+		"setup", "", "Start Jira plugin setup flow")
+	setup.RoleID = model.SystemAdminRoleId
+	withFlagInstance(setup, optInstance, routeAutocompleteInstalledInstanceWithAlias)
+	return setup
```

Comment:
In the case if the `/setup` command, we actually don't want to include the instance flag. I think this should work along with changing the calling function:

```suggestion
func createSetupCommand() *model.AutocompleteData {
	setup := model.NewAutocompleteData(
		"setup", "", "Start Jira plugin setup flow")
	setup.RoleID = model.SystemAdminRoleId
	return setup
```

Reply:
Perfect, done!